core/utils/image_urls.py

```diff
@@ -1,32 +1,20 @@
 from django.conf import settings
 from django.utils.http import urlquote
 
-# Enums / constants for telling image_server_for_page which URL prefix to use
-RESIZE = 0
-TILE = 1
-
 def thumb_image_url(page):
     return resize_url(page, settings.THUMBNAIL_WIDTH)
 
 def medium_image_url(page):
     return resize_url(page, 550)
 
 def resize_url(page, size):
-    return  "%s/full/%d,/0/default.jpg" % (image_server_for_page(RESIZE, page), size)
+    return  "%s/full/%d,/0/default.jpg" % (iiif_for_page(page), size)
 
 def specific_tile_url(page, w, h, x1, y1, x2, y2):
-
     w, h, x1, y1, x2, y2 = list(map(int, [w, h, x1, y1, x2, y2]))
-    return "%s/%d,%d,%d,%d/%d,%d/0/default.jpg" % (image_server_for_page(TILE, page), x1, y1, x2-x1, y2-y1, w, h)
-
-def image_server_for_page(server_type, page):
-    server = ""
-    if server_type == RESIZE:
-        server = settings.RESIZE_SERVER
-    if server_type == TILE:
-        server = settings.TILE_SERVER
-
-    return "%s/%s" % (server, urlquote(page.relative_image_path, safe=""))
+    return "%s/%d,%d,%d,%d/%d,%d/0/default.jpg" % (iiif_for_page(page),
+        x1, y1, x2-x1, y2-y1, w, h)
 
-def iiif_info_for_page(page):
-    return "%s" % (image_server_for_page(TILE, page))
+def iiif_for_page(page):
```

Comment:
Why the rename here?  This is a link to the IIIF Info JSON request.

Reply:
After removing the RESIZE and TILE distinction, there was no difference between the `image_server_for_page` and `image_info_for_page` functions. So I thought the name should be more generic. I guess I thought it would be confusing for `resize_url` and `specific_tile_url` to be calling `iiif_info_for_page` because I was just thinking of the function names, and I didn't think further about the specifics. :man_shrugging:

Reply:
I think renaming it to `page_iiif_info_url` might be more clear anyway. What do you think?

Reply:
Hmm, yeah, `iiif_info_for_page` is no longer really a distinct entity.  I'm not sure how I feel about that.  Probably not worth undoing, though.

Reply:
Ack, I didn't see your update when I posted mine.  Grr.

Anyway, up to you if you want to re-rename it.  I think your choice of name is indeed clearer, but is it worth your time?  I'm good either way.